### PR TITLE
add CN translation for API update

### PR DESCRIPTION
**Pull request by xrdavies**
Updated sections:

- api/connect
-
-

### CODE REVIEW

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/api/connect.md [hunk 1]
```diff
@@ -68,15 +63,19 @@ const gearApi = await GearApi.create({ customTypes: { ...yourCustomTypesExample
 async function connect() {
   const gearApi = await GearApi.create();
 
-  const [chain, nodeName, nodeVersion] = await Promise.all([
+  const [chain, nodeName, version] = await Promise.all([
```
Comment (liushooter) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/api/connect.md:66]:
```js
  const [chain, nodeName, nodeVersion] = await Promise.all([
    gearApi.chain(),
    gearApi.nodeName(),
    gearApi.nodeVersion(),
  ]);
```
The previous code worked well， you can try this code https://github.com/GearFans/example/blob/main/src/connect.ts

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/api/connect.md [hunk 2]
```diff
@@ -68,15 +63,19 @@ const gearApi = await GearApi.create({ customTypes: { ...yourCustomTypesExample
...
   ]);
-  console.log(`You are connected to chain ${chain} using ${nodeName} v${nodeVersion}`);
-
-  gearApi.gearEvents.subscribeNewBlocks((header) => {
-    console.log(`New block with number: ${header.number.toNumber()} and hash: ${header.hash.toHex()}`);
+  console.log(
+    `You are connected to chain ${chain} using ${nodeName} v${version}`,
+  );
+
+  const unsub = await gearApi.gearEvents.subscribeNewBlocks((header) => {
```
Comment (liushooter) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/api/connect.md:75]: `subscribeNewBlocks` Revised to `subscribeToNewBlocks`
 you can try this  latest version code https://github.com/GearFans/example/blob/main/src/connect.ts

Comment (xrdavies) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/api/connect.md:75]: > `subscribeNewBlocks` Revised to `subscribeToNewBlocks` you can try this latest version code https://github.com/GearFans/example/blob/main/src/connect.ts

Thanks for the comments. Pls help to try the EN version. CN version should align with the EN version.

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/api/connect.md (hunk 1 above)
Comment (xrdavies) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/api/connect.md:66]: > ```js
>   const [chain, nodeName, nodeVersion] = await Promise.all([
>     gearApi.chain(),
>     gearApi.nodeName(),
>     gearApi.nodeVersion(),
>   ]);
> ```
> 
> The previous code worked well， you can try this code https://github.com/GearFans/example/blob/main/src/connect.ts

Thanks for the comments. Pls help to try the EN version. CN version should align with the EN version.

**Review by liushooter — APPROVED** [review]

**Pull request merged**